react-electron-poc/src/renderer/components/layout/Layout.tsx

```diff
@@ -421,7 +418,10 @@ const Layout = () => {
                                 (resourceInfo) => resourceInfo.shortName,
                             ),
                         ].map((addPanelShortName) => (
-                            <option value={addPanelShortName}>
+                            <option
+                                key={addPanelShortName}
```

Comment:
I am just curious, why are the key and the value the same?

Reply:
Interestingly, `key` is a [React keyword attribute](https://reactjs.org/docs/lists-and-keys.html) that has to do with ordering repeated elements. `value` is what we might think of normally as the key. What shows up to the user is the text in the element, but the `value` is what the code uses.